becca-new-app/src/app.module.css

```diff
@@ -0,0 +1,12 @@
+body {
```

Comment:
You need to create specific class names on each element you want to style. You can't just call button, body, and h1 here. Please read this - https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/